pkg/controller/perconaservermongodb/secrets.go

```diff
@@ -158,3 +134,42 @@ func (r *ReconcilePerconaServerMongoDB) reconcileUsersSecret(ctx context.Context
 
 	return nil
 }
+
+func fillSecretData(cr *api.PerconaServerMongoDB, data map[string][]byte) (changes bool, err error) {
```

Comment:
I suggest staying away from named return types, they introduce a bunch of janky things. To see those its best to see Dave Cheney tip on them: https://dave.cheney.net/practical-go/presentations/gophercon-israel.html#_avoid_named_return_values

Reply:
Thanks for sharing https://github.com/percona/percona-server-mongodb-operator/pull/985/commits/d40a81a9166469829ee2e1a8d6df05c6a67d145b